### PR TITLE
docs: database management and desc table

### PR DESCRIPTION
**Pull request by killme2008**
Main changes:

* Adds database management doc.
* Adds `desc table` to table management doc.

### CODE REVIEW

File: docs/user-guide/table-management.md [hunk 1]
```diff
@@ -45,6 +92,26 @@ to tags in other time-series systems like [InfluxDB][1].
 
 > TBD:  Creating table in distributed mode.
```
Comment (v0y4g3r) [inline docs/user-guide/table-management.md:93]: Please also remove this line. There's no difference bwteen standalone and distributed mode regarding creating tables.

File: docs/user-guide/table-management.md [hunk 2]
```diff
@@ -43,7 +90,25 @@ to tags in other time-series systems like [InfluxDB][1].
 
 [1]: <https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#tag-key>
 
-> TBD:  Creating table in distributed mode.
+
+#### Creating a table in other database
+
+GreptimeDB doesn's support `USE [DATABASE]` statement right now, so we must use `[database].[table]` as the table name to create or manipulate a table in other database:
```
Comment (v0y4g3r) [inline docs/user-guide/table-management.md:96]:
```suggestion
GreptimeDB doesn't support `USE [DATABASE]` statement right now, so you must use `[database].[table]` as the table name to create or manipulate a table in other databases:
```

File: docs/user-guide/table-management.md [hunk 3]
```diff
@@ -77,6 +142,44 @@ show tables like monitor;
 1 row in set (0.00 sec)
 ```
 
+
+Listing Tables in other database:
```
Comment (v0y4g3r) [inline docs/user-guide/table-management.md:146]:
```suggestion
List tables in other databases:
```

File: docs/user-guide/table-management.md [hunk 4]
```diff
@@ -77,6 +142,44 @@ show tables like monitor;
...
+| Tables  |
++---------+
+| monitor |
++---------+
+1 row in set (0.01 sec)
+```
+
+### Describe Table
+
+Showing the table information in detail:
```
Comment (v0y4g3r) [inline docs/user-guide/table-management.md:164]:
```suggestion
Show table information in detail:
```

File: docs/user-guide/table-management.md [hunk 5]
```diff
@@ -43,7 +90,25 @@ to tags in other time-series systems like [InfluxDB][1].
 
 [1]: <https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#tag-key>
 
-> TBD:  Creating table in distributed mode.
+
+#### Creating a table in other database
+
+GreptimeDB doesn't support `USE [DATABASE]` statement right now, so you must use `[database].[table]` as the table name to create or manipulate a table in other databases:
```
Comment (apdong2022) [inline docs/user-guide/table-management.md:96]:
```suggestion
GreptimeDB doesn't support `USE [DATABASE]` statement at the moment, so you must use `[database].[table]` as the table name to create or manipulate a table in other databases:
```

**Review by apdong2022 — APPROVED** [review]

**Pull request merged**